packages/ui/src/utils/getAsMultiTx.ts

```diff
@@ -0,0 +1,28 @@
+import { ApiPromise } from '@polkadot/api'
+import { SubmittableExtrinsic } from '@polkadot/api/types'
+import { ISubmittableResult } from '@polkadot/types/types'
+import { Weight } from '@polkadot/types/interfaces'
+import { HexString, MultisigStorageInfo } from '../types'
+
+interface Params {
+  api: ApiPromise
+  threshold: number
+  otherSignatories: string[]
+  tx?: SubmittableExtrinsic<'promise', ISubmittableResult> | HexString
+  weight?: Weight
+  when?: MultisigStorageInfo['when']
+}
+export const getAsMultiTx = ({ api, threshold, otherSignatories, tx, weight, when }: Params) => {
+  return api.tx.multisig.asMulti.meta.args.length === 6
```

Comment:
Can we make a "6" a constant that will describe itself? as it looks as magic number now :)

Reply:
Good point, it is indeed a magic number as it's actually not future proof. Unfortunately we have no other choice. Joystream will migrate in the next months, and we can eventually remove this "if". I'll make this a constant such as `PARAM_LENGTH_OLD_ASMULTI`